### PR TITLE
KNNClassifier accepts multiple input categories

### LINKED ISSUE #2366: Update the KNNClassifier to accept multiple input categories
Enable the KNNClassifier for multi-label classification. This is a subtask of #2335.

### PR DESCRIPTION
**Pull request by BoltzmannBrain**
Fixes #2366.

### CODE REVIEW

**Comment by BoltzmannBrain** [comment]
@scottpurdy please review.

File: tests/integration/nupic/regions/multiclass_knn_test.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+import csv
+import os
+import tempfile
+import unittest2 as unittest
```
Comment (scottpurdy) [inline tests/integration/nupic/regions/multiclass_knn_test.py]: just `import unittest`

**Comment by scottpurdy** [comment]
Make sure you run pylint, otherwise :+1: - let me know when you're ready for merge

**Comment by BoltzmannBrain** [comment]
@scottpurdy thanks, ready for merge.

**Pull request merged**